tools/ci_build/github/azure-pipelines/android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml

```diff
@@ -14,8 +14,6 @@ jobs:
   - script: sudo apt-get update -y && sudo apt-get install -y coreutils ninja-build
     displayName: Install coreutils and ninja
 
-  - template: templates/set-up-gradle-wrapper-step.yml
```

Comment:
@HectorSVC FYI, this will probably fix a build failure after I removed this step template in a previous PR.